src/api/custom-schema.ts

```diff
@@ -132,6 +137,12 @@ export const extendGraphqlSchema = (schema: GraphQLSchema) =>
 
             const mappedSicCodes = mapSicCodes(company, sicCodes);
 
+            if (mappedSicCodes && oldSicCodes) {
+              await context.db.ExporterCompanySicCode.deleteMany({
+                where: [...oldSicCodes],
+              });
+            }
+
```

Comment:
Can we add some comments to this resolver now, lots going on 

Also do we need to do a `.length` check here?